src/components/Section.js

```diff
@@ -67,7 +66,7 @@ function Section(props) {
                     </View>
                 </View>
 
-                {!_.isEmpty(props.subtitle) && (
+                {!props.subtitle && (
```

Comment:
I'm not sure why you removed `!_.isEmpty(props.subtitle)` lets revert back to using that method as it's safer than inline strings

Reply:
Ah I see- probably from my comment [here](https://github.com/Expensify/App/pull/26406#discussion_r1338029531). I think we could use `Boolean(props.subtitle)` instead.

Reply:
Or `!!props.subtitle` I think also works – and is [maybe a better practice going forward](https://expensify.slack.com/archives/C0593T50UHG/p1695973319256059)

Reply:
@roryabraham I just applied georgia's suggestion 😅